src/type.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types';
+
+export const UserTypes = PropTypes.shape({
+  id: PropTypes.number.isRequired,
+  name: PropTypes.string.isRequired,
+  username: PropTypes.string.isRequired,
+  email: PropTypes.string.isRequired,
+  address: PropTypes.objectOf(PropTypes.shape({
+    street: PropTypes.string.isRequired,
+    suite: PropTypes.string.isRequired,
+    city: PropTypes.string.isRequired,
+    zipcode: PropTypes.string.isRequired,
+    geo: PropTypes.objectOf(PropTypes.shape({
+      lat: PropTypes.string.isRequired,
+      lng: PropTypes.string.isRequired,
+    })),
+  })),
+  phone: PropTypes.string.isRequired,
+  website: PropTypes.string.isRequired,
+  company: PropTypes.objectOf(PropTypes.shape({
+    name: PropTypes.string.isRequired,
+    catchPhrase: PropTypes.string.isRequired,
+    bs: PropTypes.string.isRequired,
+  })),
+});
+
+export const Types = PropTypes.shape({
```

Comment:
```suggestion
export const TodoShape = PropTypes.shape({
```
Indicate whose type this is